docker/cli/.gitconfig

```diff
@@ -0,0 +1,2 @@
+[safe]
```

Comment:
Consider writing a comment here why we want consider all directories to be safe and in which context.

Reply:
@spaceo I have tried to explain this in the commit comment. To you prefer an inline comment in the `.gitconfig` file?

Reply:
Hmm. Don't know if that is even possible. And it does also not really belong in the docker compose file. So lets keep it like this.